secboot/secboot_tpm.go

```diff
@@ -278,6 +287,30 @@ func readKeyFileImpl(keyfile string) (*sb.KeyData, *sb_tpm2.SealedKeyObject, err
 
 var readKeyFile = readKeyFileImpl
 
+// TODO: consider moving this to secboot
```

Comment:
it already is in `secboot`, unless you meant github.com/canonical/secboot

Reply:
Right. I think actually meant secboot_sb.go. Not secboot_tpm.go.